README.md

```diff
@@ -141,6 +136,10 @@ ${DREDD_EXECUTABLE} -p build/compile_commands.json math/src/*.cc --mutation-info
 The `-p` option allows the compilation database generated by CMake above to be passed to Dredd.
 This is so that Dredd knows the correct compiler options to use when processing each source file.
 
+The --mutation-info-file argument is used to specify a JSON file to which Dredd will output 
```

Comment:
Rephrase to "The optional --mutation-info-file argument". Also, please put `--mutation-info-file` in backticks (like I have done in this comment).